skype/skype-ps/skype/Grant-CsApplicationAccessPolicy.md

```diff
@@ -60,20 +60,20 @@ The command shown above assigns the per-user application access policy ASimplePo
 PS C:\> Grant-CsApplicationAccessPolicy -PolicyName "ASimplePolicy" -Global
 ```
 
-The command shown above assigns the per-user application access policy "ASimplePoicy" to all the users in the tenant, except any that have an explicit policy assignment.
+The command shown above assigns the per-user application access policy "ASimplePolicy" to all the users in the tenant, except any that have an explicit policy assignment.
 
 ## PARAMETERS
 
 ### -Identity
 
-Indicates the user (object) ID of the user account to be assigned the per-user application access policy. 
+Indicates the user (object) ID of the user account to be assigned the per-user application access policy. If this parameter is not specified, the application policy is applied to all users in the organization, except any that have an explicit policy assignment.
```

Comment:
I think there are two parameter sets on this cmdlet:
![image](https://user-images.githubusercontent.com/33589238/115606308-d3d29000-a2e3-11eb-8a0f-002d5774e8af.png)

Identity and Global cannot be used together. If we want to apply the policy to all users, we need to use the parameter Global instead of Identity. But it's not enough to not include Identity.

Reply:
Maybe these parameter sets could be "Identity (Default)" and "GrantToTenant".

Reply:
Thanks @dariomws ! Pushed suggested changes to my patch directly